### DISCUSSION TITLE
Παρουσιάσεις εξ αποστάσεως

### DISCUSSION THREAD

**Discussion by p14kairas** [courses-ionio/hci#1547] [Announcements]
Στις Απαντήσεις σε συχνές απορίες αναγράφεται:

 "Δεν προβλέπεται παράταση για κανένα παραδοτέο. Αν κάποιος/α δεν ενημερώσει την σελίδα του/της ή δεν μπορεί να βρεθεί σε κάποια ημερομηνία, τότε μπορεί είτε να παρουσιάσει στον βοηθό του μαθήματος εξ αποστάσεως είτε να έρθει στην αμέσως προηγούμενη ημερομηνία" 
 
 Θα ισχύσουν οι παρουσιάσεις σε βοηθό του μαθήματος εξ αποστάσεως η όχι?

**Comment by epidrome** [comment]
ευχαριστούμε για την παρατήρηση!

ενημερώθηκε η παράγραφος ώστε να είναι συμβατή με τους τρέχοντες κανόνες υπουργείου-ιδρύματος

https://courses-ionio.github.io/help/faq/
